examples/user-management/flutter-user-management/.env.example

```
@@ -1,2 +1,2 @@
 SUPABASE_URL=https://mysupabasereference.supabase.co
-SUPABASE_ANON_KEY=my.supabase.anon.key
\ No newline at end of file
+SUPABASE_PUBLISHABLE_KEY=my.supabase.anon.key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)